packages/js/components/src/badge/style.scss

```diff
@@ -8,6 +8,7 @@
 	font-size: 14px;
 	line-height: 27px;
 	align-items: center;
-	width: 32px;
-	height: 28px;
+	width: auto;
+	height: auto;
```

Comment:
```suggestion
    min-width: 32px;
    height: 28px;
```

It seems to cause the normal styling to be a bit different. 

![Screenshot 2023-10-06 at 16 10 18](https://github.com/woocommerce/woocommerce/assets/4344253/2592d2eb-3ab7-40c4-a8aa-4cafd3a6012b)

Fixed:
![Screenshot 2023-10-06 at 16 10 22](https://github.com/woocommerce/woocommerce/assets/4344253/9e6e34d0-47e4-4af2-bf0d-93bcbd7857bb)